Source/Tests/testBase.h

```diff
@@ -0,0 +1,45 @@
+#ifndef __testBase_h
```

Comment:
Just a note: this file should probably follow Urho standards. I.e. pragma once instead of guards, and file name matching the class name. Also, fix formatting inside. Also, license headers (or maybe they are unneeded?)